test_user_top_tracks fails for users with just one top track

### What did you do?
I forked the repository and tried running the test suite to make sure that my environment is set up correctly. I used a new Last.fm account that had no scrobbled tracks yet.

The test for test_user_top_tracks failed.

### What did you expect to happen?
Well, I expected it to succeed.

### What actually happened?
```
_________________TestPyLastUser.test_user_top_tracks _____________________________

self = <tests.test_user.TestPyLastUser testMethod=test_user_top_tracks>

    def test_user_top_tracks(self):
        # Arrange
        lastfm_user = self.network.get_user(self.username)
    
        # Act
        things = lastfm_user.get_top_tracks(limit=2)
    
        # Assert
>       self.helper_two_different_things_in_top_list(things, pylast.Track)

tests/test_user.py:223: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
tests/test_pylast.py:112: in helper_two_different_things_in_top_list
    self.assertEqual(len(things), 2)
E   AssertionError: 1 != 2
```
`things` has a length of 1 since the only scrobbles on the account are the "Test Artist — test title" scrobbles created by previous tests.

### What versions of OS, Python and pylast are you using?
Debian stable
Python 3.5.3
Current repository state for pylast

### Solution ideas
Either make sure that the test checks that `things` actually contains two things before calling `self.helper_two_different_things_in_top_list`, or scrobble a different test track before running this test so any test account will have at least two top tracks.

--- a/tests/test_user.py
+++ b/tests/test_user.py
@@ -214,7 +214,7 @@ class TestPyLastUser(PyLastTestCase):
 
     def test_user_top_tracks(self):
         # Arrange
-        lastfm_user = self.network.get_user(self.username)
+        lastfm_user = self.network.get_user("RJ")
 
         # Act
         things = lastfm_user.get_top_tracks(limit=2)

